server/routes/country.route.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+const router = express.Router();
+const countryController = require('../controllers/country.controller');
+
```

Comment:
We have a simpler syntax for defining routes, pls check the document routes for reference.

Reply:
would be great if you could share an example syntax or a link to the example, I coudn't find one